scripts/typescript-compatibility-build.yml

```diff
@@ -155,7 +155,7 @@ jobs:
 
       # Invoke POST API with json body
```

Comment:
please try to revert any changes to this file

Reply:
PR #9179 attempts to apply long term fix.

Reply:
welp it looks like this file is screwed even on master (9d0a0b4ba159b96e4dff389c6e51980421e5d006). I'll revert it to previous state.

Reply:
If you don't mind fixing the indentation, that would be great! It should stick now that Cliff's change is in.

Reply:
@ecraig12345 Just did, waiting for a good CI :)